controllers/tutorialsControllers.js

```diff
@@ -7,48 +7,52 @@ const tutorialsAll = async (req, res) => {
       createdAt: -1,
     });
     res.json(tutorials);
+    res.status(200).send();
   } catch (error) {
-    res.json({ message: error });
+    res.status(404).send();
```

Comment:
We should specify error message here.

Pattern of error should be:
```
{
  message: 'not found'
}
```

Same pattern should be applied to all the error responses.